### PR TITLE
Move archetype-python-release.yml into the public repo

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Move archetype-python-release.yml into the public repo
Add dependsOn: PublishPackage for PublishDocs and UpdatePackageVersion deployments.

This PR Depends on https://github.com/Azure/azure-sdk-tools/pull/439

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/template/azure-template/setup.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
     license='MIT License',
     # ensure that the development status reflects the status of your package
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        "Development Status :: 5 - Production/Stable",
```
Comment (praveenkuttappan) [inline sdk/template/azure-template/setup.py]: I dont think we want to make this production stable. Can you please revert this change?
